azerothcore/azerothcore-wotlk
# DB/Locales: Add missing esES & esMX quest locales(Port from TC)

## Changes Proposed:
-  Add Missing Spanish locales.

## Tests Performed:

-  Tested on windows 10 and ubuntu 20.

## How to Test the Changes:

- To test these changes you must have the client in EsMX or EsES
- .quest add ``quest_id`` verify that the texts are correctly translated and logically that the description is not empty.

## Target Branch(es):







 
## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.
